extension-dev.code-workspace

```diff
@@ -229,51 +204,22 @@
   "launch": {
     "version": "0.2.0",
     "compounds": [
-      {
-        "name": "Test Extension",
-        "configurations": [
-          "ExtensionTests",
-          "ExtensionTestRunner",
-        ],
-        "stopAll": true,
-        "presentation": {
-          "group": "test",
-          "order": 1
-        },
-        // This is here so instead of under TestRunner so that the attach doesn't start until the compile is complete
-        "preLaunchTask": "test-watch"
-      }
+      // {
+      //   "name": "Test Extension",
+      //   "configurations": [
+      //     "ExtensionTests",
+      //     "ExtensionTestRunner",
+      //   ],
+      //   "stopAll": true,
+      //   "presentation": {
+      //     "group": "test",
+      //     "order": 1
+      //   },
+      //   // This is here so instead of under TestRunner so that the attach doesn't start until the compile is complete
+      //   "preLaunchTask": "test-watch"
+      // }
```

Comment:
Did you mean to leave this commented out?

Reply:
Nope, this isn't needed as I was able to consolidate the launch and attach into a single config using simplePort. Theoretically you should be able to use NODE_OPTIONS to auto-attach but I cannot for the life of me figure it out.
ed5fc373480fbbf928e1c2baa187dba41bcd8a2e